css/index.css

```diff
@@ -0,0 +1,387 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: Poppins;
+}
+
+.wrapper {
+  width: 375px;
+  background-color: #e5e5e5;
+  display: grid;
+}
+
+.top-container {
+  padding: 2%;
+  background-color: white;
+  border-bottom-left-radius: 10%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 30%;
+  width: 85%;
+}
+
+.left-text {
+  font-weight: 700;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.top-body-background {
+  background-image: url("../assets/header-shapes\ mobile.png");
+  background-size: contain;
+  background-position: 1ch -1em;
+  background-repeat: no-repeat;
+}
+
+.top-body {
+  justify-items: center;
+}
+
+.top-body h1 {
+  margin-bottom: 0%;
+  font-size: 35px;
+  font-weight: bolder;
+  padding: 5px;
+  width: 70%;
+  color: #172b4d;
+}
+
+.top-body p {
+  width: 75%;
+  margin-bottom: 5%;
+  color: #344563;
+}
+
+.footer-connect {
+  margin-bottom: 7%;
+  color: #7f8cff;
+  text-decoration: none;
+}
+
+.footer-connect a {
+  text-decoration: none;
+  font-size: large;
+}
+
+.top-footer {
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 40%;
+  list-style: none;
+}
+
+.second-container {
+  padding-left: 3%;
+  padding-right: 3%;
+  border-radius: 0 0 0 0;
+}
+
+.second-containerl {
+  padding-left: 3%;
+  padding-right: 3%;
+  border-radius: 0 0 0 0;
+}
+
+.inner-second-container {
+  padding: 4%;
+  margin-bottom: 20%;
+  box-shadow: rgb(163, 17, 17);
+  background-color: white;
+  border-radius: 4%;
+}
+
+.inner-second-containerl {
+  padding: 4%;
+  box-shadow: rgb(163, 17, 17);
+  background-color: white;
+  border-radius: 4%;
+  margin-bottom: 100px;
+}
+
+.top-image img {
+  width: 100%;
+}
+
+.text-inner-container h2 {
+  font-weight: 800;
+  color: #344563;
+  margin-bottom: 2%;
+}
+
+.canopy-inner-container {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.canopy-inner-container #text1 {
+  padding-right: 10px;
+  font-size: 13px;
+  color: #344563;
+}
+
+.canopy-inner-container #text {
+  color: #7a869a;
+}
+
+.canopy-inner-container ul {
+  display: flex;
+  font-size: 13px;
+  justify-content: space-around;
+  width: 70%;
+}
+
+.canopy-daily {
+  margin-bottom: 20px;
+}
+
+.canopy-list {
+  display: flex;
+  justify-content: space-between;
+  width: 70%;
+}
+
+.canopy-list li {
+  background-color: #ebebff;
+  border-radius: 10px;
+  padding: 1px 5px;
+  color: #6070ff;
+  list-style: none;
+}
+
+#second-container-button {
+  border: 1px solid #396df2;
+  width: 50%;
+  border-radius: 10px;
+  margin: 20px 0;
+  padding: 15px 5px;
+  color: #396df2;
+  font-size: large;
+}
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the buttons :clap: However your buttons do not look like Figma's design (see image) and also they are not following the button styling: Active, Hover and Pressed (see design). Kindly fix that 

### Your buttons 
![Screenshot from 2023-03-27 15-02-09](https://user-images.githubusercontent.com/91142432/227982582-31d819a2-f0aa-4900-8217-93548b1d231e.png)

### What it suppose to look like
![Screenshot from 2023-03-24 18-26-25](https://user-images.githubusercontent.com/91142432/227982944-d8c2f814-9b31-4295-8a81-0f0c9f519b9e.png)

This is a requirement of the project but the previous reviewers didn't notice it. so I requested this as an optional but I highly recommend you to make the change